packages/@aws-cdk/aws-eks/test/test.helm-chart.ts

```diff
@@ -41,6 +43,59 @@ export = {
       expect(stack).to(haveResource(eks.HelmChart.RESOURCE_TYPE, { Release: 'hismostprobablylongerthanfiftythreecharacterscaf15d09' }));
       test.done();
     },
+    'should handle chart from S3 asset'(test: Test) {
```

Comment:
This also deserves an integ test. Please add add a case to https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts